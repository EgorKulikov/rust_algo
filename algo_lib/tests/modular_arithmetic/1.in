1000 3
1 / 999
1 / 998
578 * 178
13 4
7 / 9
9 * 3
0 - 9
10 + 10
0 0
