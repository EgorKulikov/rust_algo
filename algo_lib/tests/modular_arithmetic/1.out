999
-1
884
8
1
4
7
